### PR TITLE
now builds without warnings, even under -Wall

### PR DESCRIPTION
**Pull request by pnikiel**

### CODE REVIEW

File: src/nodemanagerbase.cpp [hunk 1]
```diff
@@ -184,25 +184,21 @@ UA_LocalizedText make_localised( UaString text )
...
     return out;
 }
-    
+
 UaStatus NodeManagerBase::addNodeAndReference( 
     UaNode* parent,
     UaNode* to,
     const UaNodeId& refType)
 {
-   // std::cout << __FUNCTION__ << "from.id=" << from.toString().toUtf8() << " to.id=" << to->nodeId().toString().toUtf8() <<  std::endl;
```
Comment (ben-farnham) [inline src/nodemanagerbase.cpp:193]: Add a LogIt statement here - I assume if this line was important enough to merit a std::cout it should probably also be loggable?

**Review by ben-farnham — CHANGES_REQUESTED** [review]
Apart from the LogIt comment looks good.

File: src/nodemanagerbase.cpp (hunk 1 above)
Comment (pnikiel) [reply src/nodemanagerbase.cpp:193]: Hi Ben,
I removed this line completely in this commit, I think there is nothing to fix here. I don't think it deserves a log from here.

**Comment by ben-farnham** [comment]
That was my only comment, so good to merge I think.

**Pull request merged**